### DISCUSSION TITLE
How to increase the padding value for the SegmentedControl component, for the root element

### DISCUSSION THREAD

**Discussion by dmitry-ff** [mantinedev/mantine#5083] [Q&A] [answered]
![image](https://github.com/mantinedev/mantine/assets/81219183/73b104ae-cf07-409f-b051-a5091494e054)
When you change the padding, you get the following result, has anyone encountered this, how can I fix it?

**Comment by rtivital** [comment, marked as answer]
It is a known issue that will be resolved in one of the next patches
